File: JAVA/268. Missing Number.java
class Solution {
    public int missingNumber(int[] n) {
        Arrays.sort(n);
        int i,k=-1;
        for(i=0;i<n.length;i++)
        {
            if(i!=n[i])
            {
                k=i;
                break;
            }
        }
        return(k==-1?n.length:k);
    }
}

File: C++/268. Missing Number.cpp
class Solution {
public:
    int missingNumber(vector<int>& n) {
        int i,k=-1;
        sort(n.begin(),n.end());
        for(i=0;i<n.size();i++)
        {
            if(i!=n[i])
            {k=i;
            break;}
        }
        if(k==-1)
        k=(n.size());
        return k;
    }
};